exposure-notification/src/main/resources/db/migration/R__1_default_exposure_configuration.sql

```diff
@@ -4,8 +4,8 @@ insert into en.exposure_configuration
 select * from (
     values
       (126::int,
-       '{ 1, 3, 4, 4, 6, 7, 7, 8 }'::int array[8],
-       '{ 1, 1, 1, 1, 1, 1, 1, 1 }'::int array[8],
+       '{ 1, 2, 3, 4, 5, 6, 7, 8 }'::int array[8],
+       '{ 0, 0, 0, 1, 1, 1, 1, 1 }'::int array[8],
        '{ 1, 2 ,2, 4, 6, 6, 7, 8 }'::int array[8],
        '{ 0, 2, 4, 6, 6, 7, 8, 0 }'::int array[8],
```

Comment:
This should be `0, 0, 4, …` due to the second transmission risk bucket being the 10-14 days, which should be ignored as well.

Reply:
yep, changed